pkg/pillar/cmd/volumemgr/create.go

```diff
@@ -182,22 +228,32 @@ func destroyContainerVolume(ctx *volumemgrContext, status types.VolumeStatus) (b
 	return created, filelocation, nil
 }
 
+// returns size and indicates do we need to resize disk to be at least maxsizebytes
+func checkResizeDisk(diskfile string, maxsizebytes uint64) (uint64, bool, error) {
+	vSize, err := diskmetrics.GetDiskVirtualSize(log, diskfile)
+	if err != nil {
+		return 0, false, err
+	}
+	if vSize > maxsizebytes {
```

Comment:
The old code had a check for 	if maxsizebytes == 0 { so that it didn't log.Warnf in that case.

Reply:
Corrected, sorry for missing this